cassandra/metadata.py

```diff
@@ -1150,7 +1181,8 @@ def as_cql_query(self, formatted=False):
         creations are not included).  If `formatted` is set to :const:`True`,
         extra whitespace will be added to make the query human readable.
         """
-        ret = "CREATE TABLE %s.%s (%s" % (
+        ret = "%s TABLE %s.%s (%s" % (
+            ('VIRTUAL' if self.virtual else 'CREATE'),
```

Comment:
Yeah -- I think on the ticket, folks discussed using CREATE VIRTUAL, but we [and Java](https://github.com/datastax/java-driver/blob/8be7570a3c7fbba773ae2581bbf26e8196e7d6fb/driver-core/src/main/java/com/datastax/driver/core/KeyspaceMetadata.java#L383) landed on `VIRTUAL TABLE`. This will always be generated in a comment, so it ultimately doesn't matter -- neither `CREATE VIRTUAL TABLE` nor `VIRTUAL TABLE` is real syntax.